lib/room.rb

```diff
@@ -1,12 +1,26 @@
-class Rom
-
-  def initialize(category, length, width)
+class Room
+  attr_reader :category
+  def initialize(category, length = 10, width)
     @category = category
-    @length = 10
-    @width = width
+    @length = length.to_i
```

Comment:
length is already an integer when it's passed in, so no need to convert to_i here.

Reply:
Length was passed in as a string in the first test so I did that to allow string or integer input.

  def test_it_exists
    room = Room.new(:bedroom, 10, '13')
    assert_instance_of Room, room
  end

Reply:
The second argument, the length, is 10, which is an integer.  The third argument, width, was a string, and needed to be converted.